### ISSUE TITLE
New release breaks older node versions

### ISSUE THREAD

**Issue by scottaj** [fb55/entities#200]
Hello. Not sure which package to post this on because it affects several maintained by you:

1) this one
2) https://github.com/cheeriojs/dom-renderer
and 3) https://github.com/FB55/domutils

Specifically, you made an update two days ago to this package which modernized the code to use features from newer node versions under a minor version bump. 

We have an old webpack project that is stuck on an old version of node (node 4) that broke, because it pulled in that new version transitively through the domutils package.

Is there any chance you can repackage that change as a major version bump, and republish a new 2.0 release that keeps the same compatibility?

The specific error we get is 
```
var reNonASCII = /[^\0-\x7F]/gu;
^

SyntaxError: Invalid flags supplied to RegExp constructor 'gu'
    at new RegExp (native)
    at Object.<anonymous> (/Users/ajs/campspot/platform/node_modules/html-webpack-plugin/node_modules/pretty-error/node_modules/renderkid/node_modules/css-select/node_modules/domutils/node_modules/dom-serializer/node_modules/entities/lib/encode.js:54:1)
```

**Comment by harshalpandey** [comment]
We are also facing the same issue, 3-4 days back the same react-application was working but now we are getting a similar issue in IE 11. Please help us with the solution.

**SCRIPT5017: Syntax error in regular expression** 

Please help ASAP.

**Comment by bmcclareifs** [comment]
Hi @fb55 , would you be able to triage this?

**Comment by fb55** [comment]
This is pretty unfortunate. I expected setting the compiler target in tsconfig to ES5 would lower everything – seems like that was a misjudgment. I won’t be able to investigate for the next few days, but I’m happy to review PRs.

**Comment by scottaj** [comment]
@fb55 Would you be okay with 

1) a PR that just reverts the code to 2.0.0 and bumps the package version to 2.0.3
2) another PR on top of that that restores the code changes, but publishes them as 3.0

or are you looking for a PR on the actual regex issue. I can try to look into the latter as well, but I don't know much about unicode or what is being done with that new flag.

**Comment by rgranger** [comment]
Here on Node 8.9.4. The version 2.0.1 is making our karma test bundler crash. Fixing version to 2.0.0 makes the code work again. Unfortunately we can't upgrade our Nodejs version.

**Comment by bkothe** [comment]
We've got a breaking build as well due to this change.  I think @scottaj has the right idea as to fixing the issue.

**Comment by debabrata** [comment]
Lots of our builds are failing because of this release. @scottaj's proposal sounds good to me.

**Comment by melnikaite** [comment]
I created a PR with v2.0.3
Can someone release it on npm?

**Comment by igloude** [comment]
We're blocked from releasing b/c of this. Can you review @melnikaite's PR @fb55? Sounds like a pretty big issue here.

**Comment by melnikaite** [comment]
You can temporary add `"entities": "2.0.0"` to package.json, it will not allow other dependencies to use 2.0.1+
Make sure you do fresh install without `node_modules` folder, otherwise you will have to run `npm dedupe`

**Comment by fb55** [comment]
Fixed in #221

**Comment by fb55** [comment]
Published as `2.0.3`.
